packages/devtools_shared/lib/src/memory/memory_json.dart

```diff
@@ -38,10 +38,12 @@ abstract class MemoryJson<T> implements DecodeEncode<T> {
     if (payloadVersion != version) {
       // TODO(terry): Convert Payload TBD - only one version today.
       // TODO(terry): Notify user the file is being converted.
+      print('ERROR: Unable to convert JSON memory file payload version=$payloadVersion.');
```

Comment:
change to a log message

Reply:
I've added a TODO, I'll create another PR that moves all of the config_specific/logger code into devtools_shared if we think it's important to handle this one print